temel_bilgiler/kurulumlar/web_nasıl_çalışır.md

```diff
@@ -0,0 +1,57 @@
+### Giriş
+
+Web'i nasıl programlayacağınızı anlamadan önce, web'in kendisini muhtemelen şimdi sahip olduğunuzdan daha titiz bir şekilde anlamanız gerekir. Bu kavramlar, içinde çalışacağınız ekosistem hakkında daha bütünsel bir anlayış sağlar ve diğer geliştiricilerle işiniz hakkında akıllıca konuşmanıza olanak tanır.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konuların genel bir özetini içerir.
+
+* İnternetin ne olduğunu açıklayınız.
+* Paketlerin ne olduğunu ve veri aktarımı için nasıl kullanıldığını açıklayınız.
+* Bir web sayfası, web sunucusu, web tarayıcısı ve arama motoru arasındaki farkları anlayınız.
+* İstemcinin ne olduğunu kısaca açıklayınız.
+* Sunucu nedir kısaca açıklayınız.
+* IP adreslerinin ne olduğunu açıklayınız.
+* DNS sunucularının ne olduğunu açıklayınız.
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+  1. İnternetin nasıl çalıştığına dair genel bir bakış için bu [BBC short](https://www.youtube.com/watch?v=eHp1l73ztB8) 'u izleyin.  
+  2. Mozilla'nın "İnternet nasıl çalışır?" başlıklı bu [makalesini](https://developer.mozilla.org/en-US/Learn/Common_questions/How_does_the_Internet_work) okuyun.  
```

Comment:
We should not translate the titles of English resources, as this may lead users to expect that the article is written in Turkish.

fix: 2. Mozilla'nın "How does the Internet work?" adlı bu [İngilizce makalesini](https://developer.mozilla.org/en-US/Learn/Common_questions/How_does_the_Internet_work) okuyun.